CONTRIBUTORS

```diff
@@ -61,4 +61,5 @@ Collabora
  Ricardo Cañuelo
 Dipanjan Das
 Daimeng Wang
+Jiaheng Hu
```

Comment:
I'm pretty sure you should add yourself in the Google section instead.  Also make sure your @google.com email is the one in the commits.

Reply:
> So this is for the features introduced in [torvalds/linux@e1f3219](https://github.com/torvalds/linux/commit/e1f32190cf7ddd55778b460e7d44af3f76529698).
> 
> Any plans to also add the new netlink commands?
> 
> ```
> 	TIPC_NL_KEY_SET,
> 	TIPC_NL_KEY_FLUSH,
> ```
> 
> Might need to add `CONFIG_TIPC_CRYPTO` to the syzbot config for those though.

Just added the TIPC_NL_KEY_SET and TIPC_NL_KEY_FLUSH.